src/effects/backends/audiounit/audiounitmanager.mm

```diff
@@ -6,28 +6,36 @@
 
 #include "effects/backends/audiounit/audiounitmanager.h"
 
-AudioUnitManager::AudioUnitManager(AVAudioUnitComponent* _Nullable component,
-        AudioUnitInstantiationType instantiationType)
-        : m_name(QString::fromNSString([component name])),
+AudioUnitManager::AudioUnitManager(AVAudioUnitComponent* _Nullable component)
+        : m_name(component != nil ? QString::fromNSString([component name])
+                                  : "Unknown"),
           m_isInstantiated(false) {
```

Comment:
if this occurs often, it might make sense to append a counter for if you have multiple nameless components. but I guess this is more a safety net than anything else, right?

Reply:
Yes, I don't expect this to happen usually, so this is just a fallback